ui/design/src/styles/themes/dark/colors-dark.ts

```diff
@@ -43,9 +43,11 @@ const colors: IColorsDark = {
   subtitleText: '#F4F5F7', // cadetGrey
   errorText: '#FF5050', // red
   darkText: '#47484A', // system/off white/dark
-  accentText: '#4E71FF', // blue
+  accentText: '#8B9FFF', // blue
```

Comment:
on dark mode the accent text should be the same as the accent color, I spoke to Basma about this, the different accent text color is only for light theme